sphinx-src/web-api/rdm.rst

```diff
@@ -0,0 +1,259 @@
+RDM Discovery
+#############
+
+Discover RDM fixtures and get cached device info from discovered fixtures.
+
+HTTP
+****
+
+.. _discover-http-get:
+
+GET
+===
+
+``GET /api/rdm/discovery``
+
+Returns a JSON object with the following attributes:
+
+.. list-table::
+   :widths: 5 3 8 4
+   :header-rows: 1
+
+   * - Attribute
+     - Value Type
+     - Description
+     - Value Example
+   * - ``uid``
+     - string
+     - manuId:deviceId[:subId] (Hex, 4byte:8byte[:decimal integer)
```

Comment:
This could use a bit more description. You could use a snippet/reference this definition elsewhere.